client/src/services/ModelService.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable */
```

Comment:
I think we should remove this or exclude specific issues.

Reply:
@miltone92 remove it - I may have added it to allow console.log for debugging and forgot to remove